app/javascript/__tests__/case_emancipations.test.js

```diff
@@ -60,7 +60,7 @@ describe('Function that closes the children of a given parent', () => {
   test('Closes the categoryOptionsContainer', () => {
     $(document).ready(() => {
       closeChildren(category)
-      expect(category.attr('data-is-open')).toContain('false')
+      expect(category.attr('data-is-open')).toContain('banana')
```

Comment:
@compwron 
some tests that should have failed passed when I ran `npm test` locally, so I decided to check if the failure would show up when I pushed, but it doesn't seem to show up anywhere as a failure.

How should I go about this?

Reply:
hmmmm that is concerning, but they are failing in CI... maybe your local is having caching issues
or maybe we are not running all the tests I should be

Reply:
so, I don't think `toContain` was the problem, as suggested [here](https://github.com/rubyforgood/casa/pull/4081/files/d0fe2fbc2df16fc0e4c35f7fe4db342a07ac5a04#r1002404117).

So, when I started to write the tests and ran `npm test`, they were failing when they should, but at some point, they stopped failing. Even if I comment my variables, and test for things like 'banana' it still passes.

Now it seems my case_emacipation.js stopped working (other js files work fine), even when I'm on the main branch, so I don't think any updates to the file on my branch are the problem. I think there is probably something wrong with my environment, but I'm at a loss about what it could be.

Reply:
@tcione helped me and we figured out what was wrong 😄 There are two different problems:

1- I followed the pattern from one of the test files that used `$(document).ready(() => {` inside the tests, turns out those tests aren't running. So the problem with my tests was they weren't running. There are other tests that aren't running as well. I'll open an issue explaining this better.

2- On why the js stopped working: there is a regex in `line 104` that tested only for digits in the pathname, but 
this is the pathname we get: '/casa_cases/CINA-07-1003/emancipation'. If I'm not mistaken, you talked about how the pathname was going to change last office hours. So that is why this also broke on main.

Reply:
I added two small issues for these bugs, and went ahead and fixed the regex one:
[issue#4107](https://github.com/rubyforgood/casa/issues/4107) and [issue#4108](https://github.com/rubyforgood/casa/issues/4108)/[PR for issue#4108](https://github.com/rubyforgood/casa/pull/4109)